src/resolvers/teams.js

```diff
@@ -97,20 +97,38 @@ const teamsResolvers = {
       return true;
     },
 
-    async updateTeam(root, { id, name, organization, amountRaised, itemsSold, isArchived }) {
-      const team = await models.Team.findOne({ where: { id } });
-      if (team === null) {
-        throw new Error('Team Not Found');
+    async updateTeamName(root, { id, name }) {
+      console.log(id, name);
+      try{
+        const team = await models.Team.update({name}, {
+          where: { id },
+          returning: true
+        });
+        if(team.length == 2){
+          return team[1][0].dataValues;
+        }
```

Comment:
are we not just able to return `team[1]`? or does that not give the right type

```suggestion
        return team[1]
```

Reply:
that case **_should_** never happen since it would only go there is ID lookup was somehow non-unique.